packages/dx-react-common/src/tests.testcafe.js

```diff
@@ -0,0 +1,7 @@
+fixture `React Common`
```

Comment:
There are all our demos from chart, grid, scheduler in the React Common package. Should we need testcafe tests for this package?